### PR TITLE
Auto-generate choices and coerce args for model forms including enum columns

### LINKED ISSUE #141: Auto detect DBEnum type when generating form
When using `keg_elements.db.columns.DBEnum` the standard `SelectField` that is generated by our form generator does not work.  The docstring recommends a pattern like:

```python
        option = wtforms.SelectField(
            'Option',
            choices=MyEnum.form_options()
            coerce=MyEnum.coerce
        )
```

Seems like we could do something to help the form generator do the right thing for the `SelectField` so it doesn't have to be customized.

### PR DESCRIPTION
**Pull request by bladams**
Fixes #141

### CODE REVIEW

File: keg_elements/tests/test_forms/test_form.py [hunk 1]
```diff
@@ -92,6 +92,39 @@ def test_blank_choice_coerce(self):
...
+
+    def test_db_enum_override_choices(self):
+        class ThingForm(ModelForm):
+            class Meta:
+                model = ents.Thing
+                csrf = False
+
+            class FieldsMeta:
+                __default__ = FieldMeta
+                status = FieldMeta(choices=[('1', 'Foo'), ('2', 'Bar')], coerce=str)
```
Comment (rsyring) [inline keg_elements/tests/test_forms/test_form.py:119]: Why is the `coerce=str` param needed here?  From a DX perspective, it seems unexpected.

**Review by rsyring — COMMENTED** [review]
Looks pretty good to me.  One inline comment.  Also need to document or adjust docs as appropriate.

File: keg_elements/tests/test_forms/test_form.py (hunk 1 above)
Comment (bladams) [reply keg_elements/tests/test_forms/test_form.py:119]: It isn't necessary for the form operation. I was just passing a trivial function through to make sure it overrode the default coerce method.

**Comment by rsyring** [comment]
:shipit:

**Pull request merged**